packages/apps/spaces/hooks/useContact/useAddContactTag.ts

```diff
@@ -18,8 +18,7 @@ export const useAddTagToContact = ({
   contactId: string;
 }): Result => {
   const client = useApolloClient();
-  const [addTagToContactMutation, { loading, error, data }] =
-    useAddTagToContactMutation();
+  const [addTagToContactMutation] = useAddTagToContactMutation();
```

Comment:
(optional) This could be opportunity to drop try catch approach from here and use `onError`, `onCompleted` and `update` methods from apollo